### PR TITLE
Kevin branch

### PR DESCRIPTION
**Pull request by OwlHaus**

### CODE REVIEW

File: src/Components/Views/Meals/Meals.js [hunk 1]
```diff
@@ -1,17 +1,73 @@
...
+    
     render() {
         return (
             <ScMeals>
-                This is a meals!!!
+                {this.state.pendingMeals.map(meal => (
+                    <ScMealBox>
+                        {meal.meal}<br />
+                        {meal.type}<br />
+                        {MealTimer.timeUntil(meal.time)}<br />
```
Comment (bist333) [inline src/Components/Views/Meals/Meals.js]: dont use br

File: src/persist.json [hunk 2]
```diff
@@ -1,10 +1,12 @@
 {
-  "some_key": "some_value",
-  "array": [
-    1,
-    "two"
+  "next_meal_time": 98555987345,
+  "pending_meals": [
+    {"meal": "Salvatore's", "type": "takeout", "time": 98555987345},
```
Comment (bist333) [inline src/persist.json]: please put in a real time

File: src/Components/Views/StartMeal/Shared.js [hunk 3]
```diff
@@ -19,4 +20,19 @@ export const ScMealType = styled.div`
...
\ No newline at end of file
+`;*/
+
+export const ScMealType = styled.div`
+    text-align: center;
+    padding: 5%;
+    width: 25%;
+    height 30px;
+    border-style: solid;
+    border-color: 2px;
```
Comment (bist333) [inline src/Components/Views/StartMeal/Shared.js]: 2px is not a color

Use 
border:  2px solid #color

**Review by bist333 — DISMISSED** [review]

File: src/Components/Views/StartMeal/Shared.js (hunk 3 above)
Comment (OwlHaus) [reply src/Components/Views/StartMeal/Shared.js]: fixed

File: src/persist.json (hunk 2 above)
Comment (OwlHaus) [reply src/persist.json]: changed to timestamp

File: src/Components/Views/Meals/Meals.js (hunk 1 above)
Comment (OwlHaus) [reply src/Components/Views/Meals/Meals.js]: changed to divs

**Review by lindseyferretti — APPROVED** [review]
Yeet

**Pull request merged**